dvc/info.py

```diff
@@ -32,6 +33,10 @@ def get_dvc_info():
 
     try:
         with Repo() as repo:
+            hardlink_lock = repo.config["core"].get("hardlink_lock", False)
+            updater = Updater(repo.tmp_dir, hardlink_lock=hardlink_lock)
+            updater.check()
```

Comment:
This will run when we raise exceptions as well. It should be part of the `CmdVersion`, I think.